RecoLocalCalo/HcalRecProducers/python/HBHEMethod2Parameters_cfi.py

```diff
@@ -8,21 +8,21 @@
     applyPulseJitter      = cms.bool(False),
     applyTimeSlew         = cms.bool(True),           #units
     ts4Min                = cms.double(0.),           #fC
-    ts4Max                = cms.vdouble(100.,70000.), #fC # this is roughly 20 GeV
+    ts4Max                = cms.vdouble(100.,45000.), #fC # this is roughly 20 GeV
```

Comment:
For the moment, all the channels have the same gain in MC conditions. At some point, it might be better to make this database-dependent, but it was pointed out that such a change would also impact M2 results for HB with HPDs (and HE pre-2017), since it hasn't been done that way so far.